### PR TITLE
Dayjs

### PR DESCRIPTION
**Pull request by AMess33**
bringing in dayJS library to help with displaying dates and calculating date differentials so a time duration of each session can be displayed on the page

### CODE REVIEW

File: apps/poker-roll/src/app/activeSessionComponent.jsx [hunk 1]
```diff
@@ -37,7 +34,6 @@ const ActiveSession = (props) => {
 };
 const ActiveSessionContainer = () => {
   const sessionsQuery = useGetAllSessions();
-  console.log(sessionsQuery.data);
   return sessionsQuery.data?.map((session) => {
```
Comment (aautem) [inline apps/poker-roll/src/app/activeSessionComponent.jsx:37]: Seeing an error in the console here: `Warning: Each child in a list should have a unique "key" prop.` https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key

File: apps/poker-roll/src/app/historyPage.jsx [hunk 2]
```diff
@@ -1,32 +1,53 @@
...
+  const sessionsQuery = useGetAllSessions();
 
-const History = (props) => {
+  return sessionsQuery.data?.map((session) => {
+    if (!session.endTime) {
+      return null;
+    }
+    const start = dayjs(session.startTime);
+    const end = dayjs(session.endTime);
+    const duration = end.diff(`${start}`, 'hour', true).toFixed('1');
```
Comment (aautem) [inline apps/poker-roll/src/app/historyPage.jsx]: Shouldn't need to put `start` in a template literal here, I think a lot of the dayjs examples use date strings but they can handle date objects as well.

File: apps/poker-roll/src/app/activeSessionComponent.jsx (hunk 1 above)
Comment (AMess33) [reply apps/poker-roll/src/app/activeSessionComponent.jsx:37]: i'm not getting this error here after adding a key, but i'm getting the same error in the Past Session map function

File: apps/poker-roll/src/app/historyPage.jsx (hunk 2 above)
Comment (AMess33) [reply apps/poker-roll/src/app/historyPage.jsx]: removed, still works the same as before

**Pull request merged**